git-commit: add example (#6097)
tldr-pages/tldr: pages/common/git-commit.md

- Commit only specific (already staged) files:

`git commit {{path/to/file1}} {{path/to/file2}}`

> - Commit staged files to the repository with a message while allowing the commit be empty:

Reviewer: How about just
```suggestion
- Allow empty commits:
```

Author: I don't think so. It doesn't describe it and doesn't fit with the page.

```suggestion
- Create a commit which has the ability to be empty with a message
```

Reviewer 2: That's too wordy.  How about
```suggestion
- Create an empty commit with no added, deleted, or changed files:
```

Author: It isn't mandatory for it to empty, the flag just allows it to be.

Author: If there are staged commits, it will put them in.

Reviewer: Another suggestion:
```suggestion
- Create a commit, even if there are no staged files:
```

Author: Yes thats fine

Reviewer 3: ....but surely you wouldn't use this unless you wanted an empty commit in the first place? Either of @marchersimon's suggestions would be fine.

Author: You might use it in a script thats automated.

> `git commit -m "{{message}}" --allow-empty`

Reviewer 4: Could you expand the arguments?
```suggestion
`git commit --message="{{message}}" --allow-empty`
```

Author: I understand however its an incredible annoyance. Also most users would probably remember 

`m for message`

Reviewer: I wouldn't be too sure about that. Newcomers to git prabably don't have to commit that often that typing `--message` would be a problem.

Reviewer: At least I'd remove the `=`:
```suggestion
`git commit --message "{{message}}" --allow-empty`
```

Reviewer 2: Hmm, I wonder if with such a common command as `git commit`, we should maybe keep the short options 🤔

I've never used `--message` before if I recall correctly 😅

Reviewer: It's the same situation as with #6083. Nobody would ever use `--sync --clean --clean`, but it's still good to know what they do, especially for beginners.

Author: yes but them commands are long and multiple flags, this is `m` if I was thinking for a message I would try -m.

The `S` for sync doesn't mean much to me, I'd immediately think search

Reviewer 3: I would suggest that this is a rare exception to the rule. Generally speaking you're going to be using this command a *lot*, so I'd suggest keeping the short version in this case, and maybe updating the description like we have on the `tar` page for example.

Author: > Hmm, I wonder if with such a common command as `git commit`, we should maybe keep the short options thinking
> 
> I've never used `--message` before if I recall correctly sweat_smile

You haven't used message or just always used -m?
